content/en/tracing/troubleshooting/dotnet_troubleshooting.md

```diff
@@ -0,0 +1,101 @@
+---
+title: .NET Troubleshooting
+kind: documentation
+---
+
+If your .NET application's traces are not showing up within Datadog UI after install, there are a few common troubleshooting items to look into.
+
+## .NET Framework
+
+### Compatability 
+
+The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and above. Additionally, only [supported frameworks][1] are automatically instrumented.  
+
+If you are are missing spans from a part of your application, please confirm the integration and version are supported and [open a support ticket][2] with this information provided to expedite our review.
+
+### Confirm installation 
+
+#### Datadog Agent
+
+Verify the Datadog Agent for Windows OS been installed locally. If not, visit the [Datadog Windows Agent][3] guide.
+
+#### Datadog .NET tracer
+
+Verify the Datadog .NET tracer been installed locally. If not, visit our [Datadog .NET APM release page][4].
+
+### Confirm environment variables are correct
+
+For automatic instrumentation of IIS applications, the required environment variables must be set within the `W3SVC` & `WAS` Windows registry paths. 
+
+#### Automatic Steps
```

Comment:
This isn't automatic. Consider changing the names
- Automatic steps ➡️ Repairing the .NET Tracer installation
- Manual steps ➡️ Settings the env vars manually (or "Editing the Registry" which sounds scarier so hopefully they go the repair route)